app/browser/dataset-preview.tsx

```diff
@@ -197,9 +202,10 @@ export const DataSetPreview = ({
               }&dataSource=${sourceToLabel(dataSource)}`}
               passHref
               legacyBehavior
+              target={isOdsIframe(router.query) ? "_top" : undefined}
```

Comment:
@noahonyejese I think here it needs to be `_blank`, not `_top` in order to open in a new tab 🤔